### PR TITLE
fix: camera not enabled on foreground notifications

### PR DESCRIPTION
**Pull request by santhoshvai**

### CODE REVIEW

File: packages/client/src/devices/InputMediaDeviceManager.ts
Comment (santhoshvai) [inline packages/client/src/devices/InputMediaDeviceManager.ts:1]: this is the fix in client for resume not working if disable was called before enable completed

File: packages/client/src/devices/InputMediaDeviceManager.ts [hunk 1]
```diff
@@ -78,6 +78,7 @@ export abstract class InputMediaDeviceManager<
       return;
     }
 
+    this.state.prevStatus = this.state.status;
```
Comment (santhoshvai) [inline packages/client/src/devices/InputMediaDeviceManager.ts:81]: this change is not really necessary, but spotted that prevState was never set when doing enable()

we dont have a usecase for prevState being disable and currentstate being enable yet

**Review by oliverlaz — APPROVED** [review]
looks good to me, but I'd like @myandrienko to also take a look.

File: packages/client/src/devices/InputMediaDeviceManager.ts [hunk 2]
```diff
@@ -74,10 +74,10 @@ export abstract class InputMediaDeviceManager<
    * Starts stream.
    */
   async enable() {
+    this.state.prevStatus = this.state.status;
```
Comment (myandrienko) [inline packages/client/src/devices/InputMediaDeviceManager.ts:77]: I think, you can replace that with `this.state.prevStatus = this.state.optmisticStatus`. Then the check in lines 101-108 will not be required.

I'm not 100% sure though, and the PR LGTM anyway.

**Review by myandrienko — APPROVED** [review]

**Pull request merged**